### DISCUSSION TITLE
db2 Installation - error if server named db2

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#34] [Ideas]
Finally installed db2, but one minor hiccup which arose as I had named the server db2. In install_db2.yml, the command "ps -ef | grep db2 | grep -v root" found the avahi service which was running as db2.local and the result was that the installation of db2 failed. Changing the command to "ps -ef | grep db2inst1 | grep -v root" solved the problem and db2 seems to have installed correctly.  Maybe worth changing the command to help others avoid the problem.

**Comment by dejo1307** [comment]
Thanks a lot! Will be part of the same PR I've just mentioned in a different topic, changing it now.

**Comment by dejo1307** [comment]
It's done, part of https://github.com/HCL-TECH-SOFTWARE/connections-automation/pull/35/files
